### PR TITLE
retrace: Decode more attribute formats

### PR DESCRIPTION
**Pull request by gerddie**
Decode GL_INT_2_10_10_10_REV, GL_UNSIGNED_INT_2_10_10_10_REV, and
GL_UNSIGNED_INT_10F_11F_11F_REV attribute types.

Signed-off-by: Gert Wollny <gert.wollny@collabora.com>

### CODE REVIEW

File: retrace/glstate_shaders.cpp [hunk 1]
```diff
@@ -304,6 +304,105 @@ struct VertexAttribDesc : AttribDesc
     }
 };
 
+/* code for converting fp11 and fp10 values to fp32 is taken from Mesa:
+ *
+ * The* Copyright (C) 2011 Marek Olšák <maraeo@gmail.com>
```
Comment (jrfonseca) [inline retrace/glstate_shaders.cpp:309]: It's the same license (MIT), so instead to duplicating the license, please just add `Copyright (C) 2011 Marek Olšák <maraeo@gmail.com>` to the top of this file, below Jose Fonseca.

Another alternative, is to put these helpers on their own header file.  Probably a good idea on its own right.

**Review by jrfonseca — COMMENTED** [review]
Otherwise looks good. Thanks

File: retrace/glstate_shaders.cpp (hunk 1 above)
Comment (gerddie) [reply retrace/glstate_shaders.cpp]: I did the latter and added a header file in helpers with these functions.

**Comment by jrfonseca** [comment]
I believe this was arlready merged as part of https://github.com/apitrace/apitrace/pull/771 so closing.

Sorry for the delay.  (It's only now I'm on PTO I'm finding a bit of time to catch up.)

**Pull request closed without merging**